### PR TITLE
Add the single node cluster profile

### PR DESCRIPTION
**Pull request by oceanc80**
Add the single node cluster profile and update the associated Makefile hack until https://github.com/openshift/api/pull/1282 is merged.

### CODE REVIEW

File: manifests/0000_50_cluster-platform-operator-manager_00-rukpak-bundles.crd.yaml [hunk 1]
```diff
@@ -37,10 +38,14 @@ spec:
         description: Bundle is the Schema for the bundles API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
```
Comment (tylerslaton) [inline manifests/0000_50_cluster-platform-operator-manager_00-rukpak-bundles.crd.yaml:41]: Just curious, did `make manifests` do this? It is weird it hasn't done it before.

**Review by tylerslaton — APPROVED** [review]
/lgtm

**Comment by tylerslaton** [comment]
It looks like the Makefile hack update that you mentioned didn't come through unless I'm missing something.

File: manifests/0000_50_cluster-platform-operator-manager_00-rukpak-bundles.crd.yaml (hunk 1 above)
Comment (oceanc80) [reply manifests/0000_50_cluster-platform-operator-manager_00-rukpak-bundles.crd.yaml:41]: I assume so? I didn't open any of those files myself 🤷‍♀️

**Review by timflannagan — APPROVED** [review]

**Comment by timflannagan** [comment]
/retitle Add the single node cluster profile

This is a prerequisite to payload introduction so adding the required labels

/label px-approved
/label qe-approved
/label docs-approved

**Pull request merged**